docs/deployment/pull-backup.rst

```diff
@@ -11,6 +11,13 @@ the SSH connection must be established by the client via a remote tunnel. Other
 network file systems like NFS or SMB could be used as well, but SSHFS is very
 simple to set up and probably the most secure one.
 
+.. note::
+
+    There are user reports that SSHFS can be extremely slow over high latency
+    networks. Using SSHFS also means Borg does all the checksumming over the
+    network which is also very slow. You should benchmark your connection's
+    throughput first before deploying a huge setup.
```

Comment:
hmm, this slightly misses the point.

the "checksumming" (chunking / hashing / lookup of the file contents) happens only if the files cache lookup considers the file as potentially changed (or new).

if that check works correctly (which might need fine tuning using --files-cache), it should work quite reliably. if a file is really changed, we need to read (chunk, ...) it anyway. if read speed in general is slow over sshfs, that is not a borg specific problem.

so, the reading (chunking, ...) is not the remarkable part that should be documented here, but that one needs to get the files cache working (what you already did below).